### PR TITLE
Refine type in logRequestResult documentation example #1520

### PR DESCRIPTION
**Pull request by danielkman**
This PR fixes issue #1520

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @danielkman,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by ktoso** [comment]
OK TO TEST

File: docs/src/test/scala/docs/http/scaladsl/server/directives/DebuggingDirectivesExamplesSpec.scala [hunk 1]
```diff
@@ -116,12 +116,12 @@ class DebuggingDirectivesExamplesSpec extends RoutingSpec {
...
       loggingAdapter:   LoggingAdapter,
       requestTimestamp: Long,
-      level:            LogLevel       = Logging.InfoLevel)(req: HttpRequest)(res: Any): Unit = {
+      level:            LogLevel       = Logging.InfoLevel)(req: HttpRequest)(res: RouteResult): Unit = {
       val entry = res match {
         case Complete(resp) =>
           val responseTimestamp: Long = System.nanoTime
           val elapsedTime: Long = (responseTimestamp - requestTimestamp) / 1000000
-          val loggingString = s"""Logged Request:${req.method}:${req.uri}:${resp.status}:${elapsedTime}"""
+          val loggingString = s"""Logged Request:${req.method}:${req.uri}:${resp.status}:$elapsedTime"""
```
Comment (ktoso) [inline docs/src/test/scala/docs/http/scaladsl/server/directives/DebuggingDirectivesExamplesSpec.scala]: Would have preferred for this to remain in {} as all other params here

**Comment by raboof** [comment]
Hi Daniel, thanks for your contribution! Indeed nicer than using `Any` there :)

**Pull request merged**